src/WindowsUtils/CreateProcessTest.cpp

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "OrbitBase/ExecutablePath.h"
+#include "Test/Path.h"
+#include "TestUtils/TestUtils.h"
+#include "WindowsUtils/CreateProcess.h"
+#include "WindowsUtils/ProcessList.h"
+
+namespace {
+std::filesystem::path GetTestExecutablePath() {
+  static auto path = orbit_base::GetExecutableDir() / "FakeCliProgram.exe";
+  return path;
+}
+
+}  // namespace
+
+using orbit_windows_utils::ProcessInfo;
+using orbit_windows_utils::ProcessList;
+
+TEST(CreateProcess, SuccessfulProcessCreation) {
+  constexpr const char* kArguments = "sleep_for_ms=5000";
+  auto result = orbit_windows_utils::CreateProcess(GetTestExecutablePath(), "", kArguments);
+  ASSERT_FALSE(result.has_error());
+  ProcessInfo& process_info = result.value();
+  std::unique_ptr<ProcessList> process_list = ProcessList::Create();
+  EXPECT_TRUE(process_list->GetProcessByPid(process_info.process_id).has_value());
+}
+
+TEST(CreateProcess, CommandLine) {
```

Comment:
I would add this testcase (the EXPECT_EQ for the command line) to the test above.